addons/docs/src/frameworks/vue/config.js

```diff
@@ -1,10 +1,17 @@
 /* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { render } from 'react-dom';
```

Comment:
Unused? @Aaron-Pool

Reply:
@shilman I thought React had to be in scope to use JSX? (The render is unused though, I suppose. I think these may have both auto imported when I added JSX to the file?)

Reply:
@Aaron-Pool Yeah was only referring to the `render` import